### PR TITLE
Add additional metadata fields

### PR DESCRIPTION
**Pull request by jasonLaster**
This should make it easier to group/filter these events

https://ui.honeycomb.io/replay/datasets/backend/result/7ZwetGhUrhm

### CODE REVIEW

File: src/ui/components/DevTools.tsx [hunk 1]
```diff
@@ -251,6 +251,8 @@ function _DevTools({
           workspace: recording?.workspace,
           metadata: recording.metadata && {
             testEnvironment,
+            pluginVersion: testEnvironment?.pluginVersion,
+            testRunner: testEnvironment?.testRunner
```
[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/ui/components/DevTools.tsx:255]: There's an arbitrary limit to deep the JSON can be and still be indexed. I think we could probably get rid of metadata and have the same benefit.

[the remark this answers is not in the document]
Comment (bvaughn) [reply src/ui/components/DevTools.tsx:255]: I don't think a limit to deep JSON would apply here, would it? These attributes are only 1 level deep.

[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/ui/components/DevTools.tsx:255]: Sadly we use the first two on boilerplate

Screenshot 2023-11-30 at 7 05 42 PM

[the remark this answers is not in the document]
Comment (bvaughn) [reply src/ui/components/DevTools.tsx:255]: I don't understand.

The code you have in this PR:
```js
 metadata: recording.metadata && {
   testEnvironment,
   pluginVersion: testEnvironment?.pluginVersion,
   testRunner: testEnvironment?.testRunner
}
```

Will produce one of the following:
```json
metadata:{
  testEnvironment: {
    pluginVersion: "...",
    testRunner: "...",
    ...
  },
  pluginVersion: "...",
  testRunner: "..."
}
```

```json
```

**Comment by bvaughn** [comment]
I don't think this change is necessary. I think the data it would just send the two fields redundantly, as mentioned above: https://github.com/replayio/devtools/pull/9945#discussion_r1411555297

Going to close this PR for now since it seems to have fizzled out. @jasonLaster, can we just chat in realtime about this if I'm missing something and you think it's still important to make some changes here?

**Pull request closed without merging**